tests/test_sigmaJs.py

```diff
@@ -361,17 +362,40 @@ def test_get_def_id_by_rule_id():
         e.value) == 'err- 404 rule not found!'
 
 
+def test_get_def_id_by_notselected_rule_id():
+    src = 'test_data/ssg-fedora-ds-arf.xml'
+
+    parser = tests.any_test_help.get_parser(src)
+    rule_id = 'xccdf_org.ssgproject.content_rule_ntpd_specify_remote_server'
+    with pytest.raises(ValueError) as e:
+        parser.get_def_id_by_rule_id(rule_id)
+    assert str(
```

Comment:
`str(e)` is better than `e.value`, as usage of the `value` attribute is not [documented](https://docs.python.org/3.7/library/exceptions.html#BaseException), while `str()` is.